edx_exams/apps/core/models.py

```diff
@@ -196,6 +196,17 @@ def get_attempt_by_id(cls, attempt_id):
             attempt = None
         return attempt
 
+    @classmethod
+    def get_latest_attempt_for_user(cls, user_id):
+        """
+        Return latest ExamAttempt (based on start time) associated with a given user_id
+        """
+        try:
+            attempt = cls.objects.filter(user_id=user_id).latest('start_time')
```

Comment:
some attempts may not yet have a start time. Does this make sure those won't get returned?